aws-cdk/build.gradle.kts

```diff
@@ -1,11 +1,12 @@
 plugins {
-    id("io.micronaut.build.internal.module")
+    id("io.micronaut.build.internal.aws-module")
 }
 
 val micronautVersion: String by project
+val micronautStarterVersion: String by project
 
 dependencies {
     api(libs.aws.cdk.lib)
-    api("io.micronaut.starter:micronaut-starter-api:$micronautVersion")
-    testImplementation(project(":function-aws-api-proxy"))
+    api("io.micronaut.starter:micronaut-starter-api:$micronautStarterVersion")
+    testImplementation(projects.functionAwsApiProxy)
```

Comment:
Why a different version property?

Reply:
because for the 4.0.x branch we will initially need a different version.